dailymotion-player-objc/DMPlayerViewController.m

```diff
@@ -30,9 +32,22 @@ - (void)openURLInSafari:(NSURL *)URL;
 @implementation DMPlayerViewController
 
 - (void)dealloc {
-  UIWebView *webView = (UIWebView *)self.view;
-  webView.delegate = nil;
```

Comment:
I understand correctly, that this is the part that's crashing – the fact that the cast to (UIWebview *) fails because it's not initialized?

If you have a reproducible way to force this crash, could you try

```objc
- (void)dealloc {
  if ([self.view isKindOfClass:[UIWebView class]]) {
    UIWebView *webView = (UIWebView *)self.view;
    webView.delegate = nil;
    [webView stopLoading];
  }
}
```